src/components/UserProfile/UserSettings/index.tsx

```diff
@@ -48,12 +44,12 @@ const UserSettings: React.FC = ({ children }) => {
   const settingsRoutes: SettingsRoute[] = [
     {
       text: intl.formatMessage(messages.menuGeneralSettings),
-      route: '/settings/main',
+      route: `/users/${user?.id}/settings/main`,
```

Comment:
The changes to the routes here break the profile routes.

Reply:
Could you clarify? I didn't find/notice anything broken as a result.  The paths just had to change since I updated it to use a shared component for the navigation.

Reply:
When a user navigates to their own settings from their own profile or from the user dropdown, the URL is different:

`/profile/settings/notifications`

Reply:
@sct Fixed!